libraries/AP_Devo_Telem/AP_Devo_Telem.h

```diff
@@ -18,6 +18,11 @@
 #include <AP_HAL/AP_HAL.h>
 #include <AP_SerialManager/AP_SerialManager.h>
 
+#ifndef AP_DEVO_TELEM_ENABLED
+    #define AP_DEVO_TELEM_ENABLED   0
```

Comment:
this should be enabled by default if not defined....changes in behavior are usually not allowed unless there is a good reason....can be a build option on the build server...or added to things dropped on 1MB boards potentially, but those are most likely to have Devo users rather than the higher end boards...

Reply:
This was discussed on the last DevCall when https://github.com/ArduPilot/ardupilot/pull/19227 was merged. By my understanding before merging that PR the Devo code was more or less non-functional?